docs/ldml/tr35-personNames.md

```diff
@@ -41,26 +41,929 @@ The LDML specification is divided into the following parts:
 
 ## <a name="Contents" href="#Contents">Contents of Part 8, Person Names</a>
 
-* 1 [Aa Aaaa](#AaAaaa)
-  * 1.1 [Bb Bbbb](#BbBbbb)
-* 2 [Cc Cccc](#CcCccc)
-  * 2.1 [Dd Cddd](#DdDddd)
+* 1 [CLDR Person Names](#CLDRPersonNames)
+  * 1.1 [Introduction](#Introduction)
+    * 1.1.1 [Not in scope](#not-in-scope)
+  * 1.2 [API Implementation](#APIImplementation)
+  * 1.3 [Person Name Formatting Overview](#PersonNameFormattingOverview)
+  * 1.4 [Example Usage](#ExampleUsage)
+* 2 [XML Structure](#2-xml-structure)
+  * 2.1 [personNames Element](#2-1-personnames-element)
+  * 2.2 [personName Element](#2-2-personname-element)
+  * 2.3 [nameOrderLocales Element](#2-3-nameorderlocales-element)
+  * 2.4 [foreignSpaceReplacement Element](#2-4-foreignspacereplacement-element)
+  * 2.5 [initialPattern Element](#2-5-initialpattern-element)
+    * 2.5.1 [Syntax](#syntax)
+* 3 [Person Name Object](#3-person-name-object)
+* 4 [Person Name Attributes](#4-person-name-attributes)
+  * 4.1 [order](#4-1-order)
+  * 4.2 [length](#4-2-length)
+  * 4.3 [usage](#4-3-usage)
+  * 4.4 [formality](#4-4-formality)
+* 5 [namePattern Syntax](#5-namepattern-syntax)
+  * 5.1 [Fields](#5-1-fields)
+  * 5.2 [Modifiers](#5-2-modifiers)
+* 6 [Formatting Process](#6-formatting-process)
+  * 6.1 [Derive the name locale](#6-1-derive-the-name-locale)
+  * 6.2 [Derive the formatting locale](#6-2-derive-the-formatting-locale)
+  * 6.3 [Derive the name order](#6-3-derive-the-name-order)
+  * 6.4 [Choose a personName](#6-4-choose-a-personname)
+  * 6.5 [Choose a namePattern](#6-5-choose-a-namepattern)
+  * 6.6 [Examples of choosing a namePattern](#6-6-examples-of-choosing-a-namepattern)
+    * 6.6.1 [Examples for rules 1 and 2](#examples-for-rules-1-and-2)
+    * 6.6.2 [Examples for rule 3 and the interaction between the rules](#examples-for-rule-3-and-the-interaction-between-the-rules)
+  * 6.7 [Deriving initials](#6-7-deriving-initials)
+  * 6.8 [Handling foreign names](#6-8-handling-foreign-names)
+* 7 [Sample Name](#7-sample-name)
+  * 7.1 [Syntax](#7-1-syntax)
+  * 7.2 [Expected values](#7-2-expected-values)
+* 8 [PersonName Data Interface Examples](#8-personname-data-interface-examples)
 
-## 1 <a name="AaAaaa" href="#AaAaaa">Aa Aaaa</a>
+## 1 <a name="CLDRPersonNames" href="#CLDRPersonNames">CLDR Person Names</a>
 
-Section 1 body text xxxxx.
+### 1.1 <a name="Introduction" href="#Introduction">Introduction</a>
 
-### 1.1 <a name="BbBbbb" href="#BbBbbb">Bb Bbbb</a>
+CLDR provides formatting for person names, such as John Smith or 宮崎駿. These use patterns to show how a name record (for example, from a database) should be formatted for a particular locale. Name data has fields for the parts of people’s names, such as a **given **field with a value of “Maria”, and a **surname **field value of “Schmidt”. 
```

Comment:
I think the format \*\*given \*\*field isn't interpreted right; it needs to be \*\*given\*\* field (the space moved over). Same for italics. Multiple instances, so easiest to search for (\*\*|\_\_) and adjust.

Reply:
Thanks, fixed these globally (was due to style misalignment in the GoogleDoc)